src/core_plugins/kibana/public/management/sections/objects/components/objects_table/components/relationships/__tests__/__snapshots__/relationships.test.js.snap

```diff
@@ -330,6 +338,7 @@ exports[`Relationships should render searches normally 1`] = `
               type="button"
             >
               <EuiIcon
+                aria-label="indexPatterns"
```

Comment:
Do we need to do some aria label mapping? This should ideally read `index patterns`.

Reply:
Yea sure!